int32 data
---
bool result
